operators-manual/button-functions.md

```diff
@@ -48,69 +48,69 @@ These button functions will command a switch to the corresponding flight mode. P
-- lights2_brighter: Increase RC Input channel 10 pwm by the amount configured by the *JS_LIGHTS_STEP* parameter
-- lights2_dimmer: Decrease RC Input channel 10 pwm by the amount configured by the *JS_LIGHTS_STEP* parameter
+- **lights2_cycle**: Increase or decrease RC Input channel 10 pwm by the amount configured by the *JS_LIGHTS_STEP* parameter, switches between increasing and increasing when output limits are reached
+- **lights2_brighter**: Increase RC Input channel 10 pwm by the amount configured by the *JS_LIGHTS_STEP* parameter
+- **lights2_dimmer**: Decrease RC Input channel 10 pwm by the amount configured by the *JS_LIGHTS_STEP* parameter
 
 ## Relay Control
 
-- relay_1_on: Sets the pin configured by the *RELAY_PIN* parameter to output HIGH
-- relay_1_off: Sets the pin configured by the *RELAY_PIN* parameter to output LOW
-- relay_1_toggle: Toggles the state of the pin configured by the *RELAY_PIN* parameter
+- **relay_1_on**: Sets the pin configured by the *RELAY_PIN* parameter to output HIGH
+- **relay_1_off**: Sets the pin configured by the *RELAY_PIN* parameter to output LOW
+- **relay_1_toggle**: Toggles the state of the pin configured by the *RELAY_PIN* parameter
 
-- relay_2_on: Sets the pin configured by the *RELAY_PIN2* parameter to output HIGH
-- relay_2_off: Sets the pin configured by the *RELAY_PIN2* parameter to output LOW
-- relay_2_toggle: Toggles the state of the pin configured by the *RELAY_PIN2* parameter
+- **relay_2_on**: Sets the pin configured by the *RELAY_PIN2* parameter to output HIGH
+- **relay_2_off**: Sets the pin configured by the *RELAY_PIN2* parameter to output LOW
+- **relay_2_toggle**: Toggles the state of the pin configured by the *RELAY_PIN2* parameter
 
-- relay_3_on: Sets the pin configured by the *RELAY_PIN3* parameter to output HIGH
-- relay_3_off: Sets the pin configured by the *RELAY_PIN3* parameter to output LOW
-- relay_3_toggle: Toggles the state of the pin configured by the *RELAY_PIN3* parameter
+- **relay_3_on**: Sets the pin configured by the *RELAY_PIN3* parameter to output HIGH
+- **relay_3_off**: Sets the pin configured by the *RELAY_PIN3* parameter to output LOW
+- **relay_3_toggle**: Toggles the state of the pin configured by the *RELAY_PIN3* parameter
 
 ## Servo Control
 
-- servo_1_inc: Increase *Aux1* servo output by 50 pwm
-- servo_1_dec: Decrease *Aux1* servo output by 50 pwm
-- servo_1_min: Set *Aux1* servo output to the configured minimum pwm according to SERVO9_MIN
-- servo_1_max: Set *Aux1* servo output to the configured maximum pwm according to SERVO9_MAX
-- servo_1_center: Set *Aux1* servo output to the configured center according to SERVO9_TRIM
+- **servo_1_inc**: Increase *Aux1* servo output by 50 pwm
+- **servo_1_dec**: Decrease *Aux1* servo output by 50 pwm
+- **servo_1_min**: Set *Aux1* servo output to the configured minimum pwm according to SERVO9_MIN
```

Comment:
Since all paramters are cover with *foo*, should we also move this parameters to cover it, like: *SERVO9_MIN* ?

Reply:
good catch, updated.